From fa73b3b976a8aec415006d7fc99bab84d04650e4 Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Wed, 19 Nov 2014 10:53:29 +0100
Subject: Fix configuration with ligbnome-desktop

A typo caused fail of configuration with enabled
support for libgnome-desktop.

https://bugzilla.gnome.org/show_bug.cgi?id=733721

diff --git a/configure.ac b/configure.ac
index dd4bcab..0ae92b1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -313,7 +313,7 @@ if test "$enable_gnome_desktop" != "no"; then
       PKG_CHECK_MODULES([LIBGNOME_DESKTOP], [gnome-desktop-3.0], has_libgnome_desktop=yes, has_libgnome_desktop=no)
    else
       PKG_CHECK_MODULES([LIBGNOME_DESKTOP], [gnome-desktop-3.0])
-      has_libgnome_desktop = yes
+      has_libgnome_desktop=yes
    fi
 
    if test x$has_libgnome_desktop = xyes; then
-- 
cgit v0.10.1

